prince_cr/_deprecated/util.py

```diff
@@ -0,0 +1,80 @@
+import numpy as np
```

Comment:
Ok, yes I think that git did not recognize this file was moved.
I will fix that